buildSrc/src/main/kotlin/kotlinx/html/generate/humanizer.kt

```diff
@@ -53,7 +53,7 @@ fun humanizeJoin(parts: Iterable<String>, separator: String): String {
         filteredParts.add(cutPart)
     }
 
-    return filteredParts.joinToString(separator = separator, transform = String::capitalize) + trailingParts.joinToString("", transform = String::capitalize)
+    return filteredParts.joinToString(separator = separator) { it.capitalize() } + trailingParts.joinToString("") { it.capitalize() }
```

Comment:
This change was made since generation fails with exception:
```
* What went wrong:
Execution failed for task ':generate'.
> 'void kotlin.jvm.internal.FunctionReferenceImpl.<init>(int, java.lang.Class, java.lang.String, java.lang.String, int)'
```